src/components/calculator-components/quotes.js

```diff
@@ -0,0 +1,39 @@
+import React, { useState, useEffect } from 'react';
+
+const Quotes = () => {
+  const category = 'money';
+  const [quotes, setQuotes] = useState([]);
+
+  useEffect(() => {
+    const fetchData = async () => {
+      try {
+        const response = await fetch(`https://api.api-ninjas.com/v1/quotes?category=${category}`, {
+          headers: { 'X-Api-Key': 'cw1sCZ0TspAVMmL1VYR95A==qPbxZI6CH7KNYcWy' },
+          contentType: 'application/json',
+        });
+        if (!response.ok) {
+          throw new Error('Network response was not ok');
+        }
+        const data = await response.json();
+        setQuotes(data);
+      } catch (error) {
+        return error;
+      }
+      return null;
+    };
+    fetchData();
+  }, [category]);
+
+  return (
+    <div>
+      {quotes.map((quote) => (
+        <div className="quote-container" key={quote}>
+          <p className="quote"><q>{quote.quote}</q></p>
+          <p className="author">{quote.author}</p>
+        </div>
+      ))}
+    </div>
+  );
+};
+
+export default Quotes;
```

Comment:
- [ ] Based on project requirements, you should have a `loading` state and `error` state.
![image](https://user-images.githubusercontent.com/84156714/235026264-6251eb3e-e046-4816-90c2-cb432839ecb8.png)

Something like this 👇🏾 

```jsx
function Quotes() {
  const [quotes, setQuotes] = useState([])
  const [hasError, setHasError] = useState(false)
  const [isLoading, setIsLoading] = useState(false)

  useEffect(() => {
    const fetchQuotes = async () => {
      setIsLoading(true)
      try {
        // Make the API request
        // Set isLoading state to true

        // If the request is successful:
          // Parse the response as JSON
          // Update the quotes state with the fetched quotes

        // If there's an error:
          // Set hasError state to true

        // Set isLoading state to false
      } catch (error) {
        setHasError(true)
        setIsLoading(false)
      }
    }

    fetchQuotes()
  }, [])

  if (hasError || Number(quotes.length) === 0)
    return <div>Something went wrong!</div>

  if (isLoading) return <div>Loading...</div>

  return (
    <ul>
      {/* Render the quotes */}
    </ul>
  )
}

export default Quotes
```


Please implement this